diff --git a/node_modules/@ucp-js/sdk/dist/esm/index.js b/node_modules/@ucp-js/sdk/dist/esm/index.js
index 8248c49..c967b78 100644
--- a/node_modules/@ucp-js/sdk/dist/esm/index.js
+++ b/node_modules/@ucp-js/sdk/dist/esm/index.js
@@ -1,2 +1,2 @@
-export * from './src/extensions';
-export * from './src/spec_generated';
+export * from './src/extensions.js';
+export * from './src/spec_generated.js';
diff --git a/node_modules/@ucp-js/sdk/dist/esm/src/extensions.js b/node_modules/@ucp-js/sdk/dist/esm/src/extensions.js
index 2d6b689..615232f 100644
--- a/node_modules/@ucp-js/sdk/dist/esm/src/extensions.js
+++ b/node_modules/@ucp-js/sdk/dist/esm/src/extensions.js
@@ -1,5 +1,5 @@
 import { z } from "zod";
-import { CheckoutCreateRequestSchema, CheckoutResponseSchema, CheckoutUpdateRequestSchema, CheckoutWithBuyerConsentCreateRequestSchema, CheckoutWithBuyerConsentResponseSchema, CheckoutWithBuyerConsentUpdateRequestSchema, CheckoutWithDiscountCreateRequestSchema, CheckoutWithDiscountResponseSchema, CheckoutWithDiscountUpdateRequestSchema, CheckoutWithFulfillmentCreateRequestSchema, CheckoutWithFulfillmentResponseSchema, CheckoutWithFulfillmentUpdateRequestSchema, OrderSchema, PaymentCredentialSchema, } from "./spec_generated";
+import { CheckoutCreateRequestSchema, CheckoutResponseSchema, CheckoutUpdateRequestSchema, CheckoutWithBuyerConsentCreateRequestSchema, CheckoutWithBuyerConsentResponseSchema, CheckoutWithBuyerConsentUpdateRequestSchema, CheckoutWithDiscountCreateRequestSchema, CheckoutWithDiscountResponseSchema, CheckoutWithDiscountUpdateRequestSchema, CheckoutWithFulfillmentCreateRequestSchema, CheckoutWithFulfillmentResponseSchema, CheckoutWithFulfillmentUpdateRequestSchema, OrderSchema, PaymentCredentialSchema, } from "./spec_generated.js";
 export const ExtendedPaymentCredentialSchema = PaymentCredentialSchema.extend({
     token: z.string().optional(),
 });
